libr/core/cmd_type.c

```diff
@@ -745,7 +745,10 @@ static int cmd_type(void *data, const char *input) {
 			r_core_cmd0 (core, "k anal/cc/*");
 			break;
 		case 0:
-			r_core_cmd0 (core, "k anal/cc/*~=cc[0]");
```

Comment:
you are breaking current tc, without changing the help message or providing an alternative command to achieve the listing o f calling conventions. i think there's another issue for this

Reply:
i think tc is already  in afcl, so i think this PR is fine, but you should fix the help message first

Reply:
Fixed.